yaksh/templates/yaksh/course_detail.html

```diff
@@ -49,7 +88,46 @@
             <div class="row">
                 <div class="col-md-4" style="padding-top:10px">
                 <input type="checkbox" name="check" value="{{ rejected.id }}">&nbsp;
-                <a href="#" data-toggle="tooltip" title=" Full Name: {{ rejected.get_full_name }}&#013;Institute: {{ rejected.profile.institute }}&#013;Department: {{ rejected.profile.department}}&#013;Roll Number: {{ rejected.profile.roll_number }}">&nbsp;{{ rejected.username }}</a>
+                <a href="#" data-toggle="modal" data-target="#reject_detail_{{forloop.counter}}">
+                {{ rejected.username }}</a>
+                    <div class="modal fade" id="reject_detail_{{forloop.counter}}" role="dialog">
+                    <div class="modal-dialog">
+                    <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal">&times;</button>
+                        <h4 class="modal-title">Information for {{ rejected.username }} </h4>
+                    </div>
+                    <div class="modal-body">
+                        <table>
+                            <tr>
+                                <td><strong>Full Name :</strong></td>
```

Comment:
This modal block is repeated above and here, can this not be made common somewhere?